### PR TITLE
[DFFE-11] Sign-Up Form Validation

### PR DESCRIPTION
**Pull request by saidatatanda**
I validated my sign-up form using javascript and edited my sign-up html and css files.
image

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]
You didn't do criteria number `16` of your task which state: `If the values in 15 above don't match, the message displayed should be "Your passwords don't match"`

**Comment by saidatatanda** [comment]
I have added the confirm password Validation to display passwords don't match
image

File: submissions/saidatAtanda/dom/signup.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        return true;
+    }
+}
+function validateForm(e) {
+    fieldValidation(firstName, nameRegex, "Please enter your First Name", "Please enter only letters",  e);
+    fieldValidation(lastName, nameRegex, "Please enter your Last Name", "Please enter only letters", e);
+    fieldValidation(phoneNumber, phoneNumberRegex, "Please enter your Phone Number", "Please enter a valid phone number", e);
+    fieldValidation(emailAddress, emailAddressRegex, "Please enter your Email Address", "Please enter a valid email", e);
+    fieldValidation(password, passwordRegex, "Please enter your Password", "Your password should have a minimum of 6 characters, 1 capital letter,1 special character eg @ and 1 number.", e);
+    confirmPasswordValidation(confirmPassword, "Please confirm your Password", "Your passwords don't match", e);
```
Comment (adaugochi) [inline submissions/saidatAtanda/dom/signup.js:47]: This function is only used once, there is no need to pass all these parameters to it, except for `e`

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by saidatatanda** [comment]
I have removed the parameters passed to the function

**Pull request merged**